designer/client/components/graph/node-modal/AdditionalProperty.tsx

```diff
@@ -51,7 +54,7 @@ export default function AdditionalProperty(props: Props) {
       showValidation={showValidation}
       //AdditionalProperties do not use any variables
       variableTypes={{}}
-      errors={propertyErrors}
+      validators={[validator]}
```

Comment:
I'd wrap whole array and all connected vars with useMemo

Reply:
:+1: thx